src/components/Charts.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from "react";
+import { Row, Col, Container } from "reactstrap";
+import Piechart from "./Piechart";
+import Barchart from "./Barchart";
+
+class Charts extends Component {
+  state = {};
```

Comment:
Will you have a state? Will you fetch some data here or will you display data passed down in props?

Reply:
There is a state in your `Dashboard` component so you will probably fetch data there and pass them down here. In that case, you can make this `Chart` a dumb component as well.